pkg/stub/handler.go

```diff
@@ -327,6 +337,75 @@ func (h *Handler) updateCfgArch(cfg *v1.Config) *v1.Config {
 	return cfg
 }
 
+func (h *Handler) imageConfigBlocksImageStreamCreation(name string) bool {
```

Comment:
Is it documented anywhere what order these rules are processed in? Does being specifically allowed override being specifically denied? Does a full match of an allowed registry override a partial match of a blocked registry, etc?

Reply:
order does not matter .... a problem with any of them, we punt

even if they did something like putting something in both allowed and blocked ... too bad, we punt until you fix your config so it makes sense

in any event, nuances like that should be handled by the owners of the config api itself ... they should have validation to catch that 

ultimately the doc updates I'll work out with @rolfedh will get into that